da-assistant/DA/Sdk/Cli/Command.hs

```diff
@@ -114,6 +114,7 @@ actionParser = subparser
   <> subcommand "stop" (Normal . Stop <$> servicesParser) "Stop a given service"
   <> subcommand "feedback" (pure $ Normal SendFeedback) "Send us feedback!"
   <> subcommand "studio" (pure $ Normal Studio) "Start DAML Studio in the current project"
+  <> subcommand "migrate" (pure $ Normal Migrate) "Migrate project config to new daml-assistant"
```

Comment:
Can we move this to the hidden command list for now, as we decide how to do migration?